src/python/pants/backend/codegen/grpcio/python/grpcio_run.py

```diff
@@ -45,6 +47,12 @@ def execute_codegen(self, target, target_workdir):
       if exit_code != 0:
         raise TaskError('{} ... exited non-zero ({}).'.format(cmdline, exit_code),
                         exit_code=exit_code)
+      # Create __init__.py in each subdirectory of the target directory so that setup_py recognizes
+      # them as modules.
+      target_workdir_obj = Path(target_workdir)
+      sources = [str(p.relative_to(target_workdir)) for p in target_workdir_obj.rglob("*.py")]
```

Comment:
The arg to `p.relative_to` can be a `pathlib.Path` object as well, like `target_workdir_path`.

Reply:
Fixed the name and passed the `target_workdir_path` object to `relative_to()`